docs/content-crag/concepts/assets/asset-materializations.mdx

```diff
@@ -261,41 +261,44 @@ class PandasCsvIOManagerWithOutputAssetPartitions(IOManager):
+When an op output is linked to an <PyObject object="AssetKey"/>, Dagster can automatically generate lineage information that describes how this asset relates to other output-linked assets.
 
-As a simplified example, imagine a two-solid pipeline that first scrapes some user data from an API, storing it to a table, then trains an ML model on that data, persisting it to a model store:
+As a simplified example, imagine a two-op job that first scrapes some user data from an API, storing it to a table, then trains an ML model on that data, persisting it to a model store:
 
-```python file=/concepts/assets/materialization_pipelines.py startafter=start_pipeline_0 endbefore=end_pipeline_0
-from dagster import solid, pipeline, OutputDefinition, AssetKey
+```python file=/concepts/assets/materialization_jobs.py startafter=start_job_0 endbefore=end_job_0
+from dagster import op, AssetKey, Out, graph
 
 
-@solid(output_defs=[OutputDefinition(asset_key=AssetKey("my_db.users"))])
+@op(out=Out(asset_key=AssetKey("my_db.users")))
 def scrape_users():
     users_df = some_api_call()
     persist_to_db(users_df)
     return users_df
 
 
-@solid(output_defs=[OutputDefinition(asset_key=AssetKey("ml_models.user_prediction"))])
+@op(out=Out(asset_key=AssetKey("ml_models.user_prediction")))
 def get_prediction_model(users_df):
     my_ml_model = train_prediction_model(users_df)
     persist_to_model_store(my_ml_model)
     return my_ml_model
 
 
-@pipeline
-def my_user_model_pipeline():
+@graph
+def my_user_model():
     get_prediction_model(scrape_users())
+
+
+my_user_model_job = my_user_model.to_job()
 ```
 
 In this case, it's certainly fair to say that this ML model, which we have assigned the key `ml_models.user_prediction`, _depends on_ the table that we created, `my_db.users` (it uses the data in the table to train the model).
 
-Why is that? By specifying the structure of your pipeline, you have already defined data depedencies between these solids. By linking the output of `scrape_users` to the input of `get_prediction_model`, we can now infer that whatever this second solid outputs will be some function of its input. Furthermore, since we have linked each of these outputs to external assets, we can use this knowledge to say that the asset associated with the output of `get_prediction_model` depends on the asset associated with the output of `scrape_users`.
+Why is that? By specifying the structure of your job, you have already defined data depedencies between these ops. By linking the output of `scrape_users` to the input of `get_prediction_model`, we can now infer that whatever this second op outputs will be some function of its input. Furthermore, since we have linked each of these outputs to external assets, we can use this knowledge to say that the asset associated with the output of `get_prediction_model` depends on the asset associated with the output of `scrape_users`.
```

Comment:
I think "graph" would make more sense than "job" in this sentence.  The graph is what specifies the structure.

Reply:
I think given the `@job` changes, might make sense to keep it as job now?

Reply:
I agree